package.json

```diff
@@ -6,9 +6,10 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "clean": "rimraf dist",
-    "prepublish": "npm run clean && npm run build",
+    "prepublish": "npm run clean && npm run build:bundles && npm run build",
```

Comment:
seems like it will run npm run build twice in same directory?

Reply:
Fixed.